### PR TITLE
refactor schedules that use Monoid to use Semigroup instead

### PR DESCRIPTION
**Pull request by SolviQorda**
as per issue #42 - hope that this is ok!

### CODE REVIEW

File: rhine/src/FRP/Rhine/Clock/Select.hs [hunk 1]
```diff
@@ -38,15 +39,15 @@ instance (Monad m, Clock m cl) => Clock m (SelectClock cl a) where
 
 
 -- | A universal schedule for two subclocks of the same main clock.
---   The main clock must be a monoid (e.g. a singleton).
+--   The main clock must be a Semigroup (e.g. a singleton).
```
Comment (turion) [inline rhine/src/FRP/Rhine/Clock/Select.hs]: Ideally, make a haddock link, i.e. `'Semigroup'`

Comment (SolviQorda) [reply rhine/src/FRP/Rhine/Clock/Select.hs]: Cool ok! Should I do the PR again, or is this just for future reference?

Comment (turion) [reply rhine/src/FRP/Rhine/Clock/Select.hs]: For the real experience, push one commit to your fork that fixes it!

Comment (SolviQorda) [reply rhine/src/FRP/Rhine/Clock/Select.hs]: haha ok :)

**Comment by turion** [comment]
Thanks!

**Pull request merged**